_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -123,12 +123,63 @@ changes to popular Bitcoin infrastructure software.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jnewbery or jonatack
+[BIP-325: Signet][review club #18267] is a PR ([#18267][Bitcoin Core #18267]) by
+Kalle Alm that implements a new kind of Bitcoin test network. Signet has since
+been merged (see [Newsletter #117][news117 signet]) for the upcoming v0.21
+release.
+
+The review club discussion covered general concepts before diving into the
+deeper technical aspects. Participants with good answers were rewarded with
+signet coins. Here is a mini-quiz on general signet concepts:
 
 {% include functions/details-list.md
-  q0="FIXME"
-  a0="FIXME"
-  a0link="https://bitcoincore.reviews/19339#l-FIXME"
+  q0="What is signet?"
+  a0="Signet is defined by [BIP325][bip325] and is both a mechanism to build
+      stable, centralized, custom proof-of-work networks that anyone may run and
+      the name of a specific global testnet."
+  a0link="https://bitcoincore.reviews/18267#l-94"
+
+  q1="Is signet intended to replace existing Bitcoin testing networks like
+      testnet or regtest?"
+  a1="They are complementary. Signet was conceived as an centralized, stable
```

Comment:
s/an/a/

Reply:
You catch this type of mistake in my content so often that it's nice for me to know you're not immune to it yourself.  :-)

Reply:
writing and editing are two separate hats! or before/after pushing hehe

Reply:
done